curiouslearning/FeedTheMonsterJS
# fix: audio issue when dragging (FM-313)

# Changes
- Fix the audio drag effect when dragging on the screen while the stone is not yet available

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any levels
- While stone is not yet loaded try to drag the screen
- Make sure audio drag effect is not playing while stone is not visible

Ref: [FM-313](https://curiouslearning.atlassian.net/browse/FM-313)


[FM-313]: https://curiouslearning.atlassian.net/browse/FM-313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
